pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "trtools"
+authors = [
+    {name = "Melissa Gymrek", email = "mgymrek@ucsd.edu"},
+]
+description = "Toolkit for genome-wide analysis of STRs"
+readme = "README.rst"
+requires-python = ">=3.5"
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3.5",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+dependencies = [
+    "cyvcf2",
+    "matplotlib",
+    "numpy",
+    "pandas",
+    "pybedtools",
+    "pysam",
+    "scikit-learn",
+    "scipy",
+    "statsmodels",
+]
+dynamic = ["version"]
+
+[tool.setuptools]
+packages = ["trtools"]
+script-files = ["trtools/testsupport/test_trtools.sh", "scripts/trtools_prep_beagle_vcf.sh"]
+license-files = ["LICENSE.txt"]
+
+[tool.setuptools_scm]
+version_file = "trtools/version.py"
```

Comment:
It seems like this file has been deleted. So is this line wrong?

Reply:
actually, setuptools will use this path to figure out how to generate a `version.py` file for us at the moment when we build the tool

so this should be fine as it is